src/views/settings/trustee-addresses.tsx

```diff
@@ -34,7 +34,7 @@ type InputProps = {
 	updateSettings: (arg: UpdateSettingsProps) => void;
 };
 
-const NonSupportedCharacters = /[!#$%^&*()_+\-=[\]{};':"\\|,.<>/?|/^\s*$/]+/;
+const NonSupportedCharacters = /[!#$%^&*()_+\-=[\]{};':"\\|,<>/?|/^\s*$/]+/;
```

Comment:
The dash should be an allowed character

Reply:
Yes you are correct, thanks for pointing it out.Is there any other character that should also be allowed ?

Reply:
Beside that character everything seems ok